app/scripts/GenomePositionSearchBox.jsx

```diff
@@ -839,7 +839,7 @@ class GenomePositionSearchBox extends React.Component {
             </div>
           )}
           renderMenu={this.handleRenderMenu.bind(this)}
-          value={this.positionText}
+          value={this.state.selectedAssembly ? this.positionText : ''}
```

Comment:
should the fallback be `"(none)"` like above?

Reply:
It depends what we want to show to the user in that case. Nothing (i.e. "") or "(none)". Maybe something more informative like "No valid assembly selected" would be better?